MultiTable/src/main.js

```diff
@@ -232,13 +257,21 @@ function bindContentToModal(modal, modalTemplate, context, content) {
   const str = modalTemplate(context);
   modal.innerHTML = '';
   modal.insertAdjacentHTML('beforeend', str);
+
+  // add event handlers to detect clicking of links
+  modal.querySelectorAll('a.list-group-item').forEach((link) => {
+    link.addEventListener('click', (evt) => {
+      trackClick('Outbound Link', evt.target.href);
+    });
+  });
 } // bindContentToModal
 
 function setSampleListeners(subjectGroup) {
   const samples = document.getElementById(subjectGroup.key)
     .querySelectorAll('.sample');
   samples.forEach((sample) => {
     sample.addEventListener('click', (evt) => {
+      trackClick('click sample', evt.target.id);
```

Comment:
consistent use of title case or all lowercase across all trackClick names being assigned